openlibrary/plugins/openlibrary/js/add_new_field.js

```diff
@@ -14,7 +14,7 @@
  *     - On cancel/close:
  *         - value of the select is set to "" to select "select xxx"
  */
-;(function($){
```

Comment:
probably don't want to fix these just yet.. 
we are blindly concatenating JS so removing this could throw errors (hence why it's warning and not erroring right now)

Imagine what happens if you concatenate

```
setTimeout()
```
with 
```
(function(){})
```
😨

Reply:
okay, I have added `;` back and updated the PR.